config/webpack.demo.js

```diff
@@ -0,0 +1,182 @@
+const helpers = require('./helpers');
+const webpack = require('webpack');
+const path = require('path');
+const sass = require('./sass');
```

Comment:
Is this repo going to use the SASS version of PatternFly, or stick with the default LESS version from V3?